cmd/utils/flags.go

```diff
@@ -289,6 +289,11 @@ var (
 		Usage:    "Manually specify optimism",
 		Category: flags.EthCategory,
 	}
+	OverrideL1ArchiveNodeRPC = &cli.StringFlag{
+		Name:     "override.l1archivenoderpc",
```

Comment:
The `override` namespace is for flags that override values hardcoded in the software, this doesn't necessarily feel like it belongs there. There is a flag for the sequencer RPC url, it should be defined with a name similar to that